spec/models/merchant_spec.rb

```diff
@@ -1,7 +1,31 @@
 require 'rails_helper'
 
 RSpec.describe Merchant, type: :model do
-  fixtures :merchants,  :customers, :transactions, :invoices
+  fixtures :merchants, :invoices, :items, :invoice_items, :transactions, :customers
+
+  scenario "finds the total revenue for a specifc merchant across all transactions" do
+    merchant = create(:merchant)
+    invoice1 = create(:invoice, merchant: merchant)
+    invoice2 = create(:invoice, merchant: merchant)
+
+    transaction1 = create(:transaction, result: "success", invoice: invoice1)
+    transaction1 = create(:transaction, result: "failed", invoice: invoice2)
+
```

Comment:
I think with factory girl, you can specify the amount of times that you want to it make an object .. something like this : lake_1, lake_2, lake_3 = create_list(:lake, 3)